utilities/getChores.js

```diff
@@ -39,7 +39,8 @@ const getChores = async () => {
 
 const getTodaysChores = async () => {
   const today = moment().format("YYYY-MM-DD");
-  const todaysChores = await getChores().map(c => c[0] === today ? c[1] : null);
+  let todaysChores = await getChores();
+  todaysChores = todaysChores.map(c => c[0] === today ? c[1] : null);
```

Comment:
Can these two lines be combined?

Reply:
I tried to do this. However I kept getting the following error: 
![image](https://user-images.githubusercontent.com/21091716/67341121-04721780-f4fd-11e9-8b74-e331fc24cbdf.png)

If there is a way to prevent that, I would be happy to know

Reply:
This actually might be the only way to do it because you need to `await` the return of `getChores()` before you do something with it. @justetz, do you know of a way to combine these lines?

Reply:
Try:

`const todaysChores = (await getChores()).map(c => c[0] === today ? c[1] : null);`

Reply:
Resolved.